server/logic/data_parser.js

```diff
@@ -54,6 +54,13 @@ function parse (data, delimiter) {
     errorStr = 'Duplicate team number found in CSV. Aborting import.'
     MsLogger.warn(errorStr)
   }
+  const doesNotHaveTeamNumber = teamNumbersArray.some(team => {
+    return team == '' || team == undefined
+  })
+  if (doesNotHaveTeamNumber) {
+    errorStr = 'Some teams are missing team numbers, Aborting import.'
```

Comment:
Looks good.
Please replace comma after "team numbers" with a period.

Reply:
@alanggreen Sorry for the long response. fixed now